### PR TITLE
Add drop for talk_to_your_factory demo

### PR DESCRIPTION
**Pull request by chriscrcodes**
Talk to Your Factory: Optimizing Factory Operations with Generative AI

### CODE REVIEW

**Comment by chriscrcodes** [comment]
> @chriscrcodes please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

File: drops/talk_to_your_factory.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "Bash"
+  ],
+  "Products": [
+    "Azure IoT Operations",
+    "Arc",
+    "Azure Arc-enabled Kubernetes",
+    "Microsoft Fabric",
+    "Azure OpenAI Service",
+    "Streamlit"
+  ]
```
Comment (fcabrera23) [inline drops/talk_to_your_factory.json:26]: Consider merging Arc and Arc-enabled Kubernetes - So many products may shown a bit ouf the card

File: drops/talk_to_your_factory.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+}
```
Comment (fcabrera23) [inline drops/talk_to_your_factory.json:27]: Missing fields: Topics,LastModified,CreatedDate should be in JSON - Ensure to include valid dates

**Review by fcabrera23 — CHANGES_REQUESTED** [review]
Thanks @chriscrcodes - Please review and let me know if you have any questions

File: drops/talk_to_your_factory.json (hunk 1 above)
Comment (chriscrcodes) [reply drops/talk_to_your_factory.json]: Done, thanks.

File: drops/talk_to_your_factory.json (hunk 2 above)
Comment (chriscrcodes) [reply drops/talk_to_your_factory.json]: Added fields: Topics,LastModified,CreatedDate (required is set to "No" in the [documentation](https://github.com/Azure/arc_jumpstart_drops/blob/main/SCHEMA.md)).

File: drops/talk_to_your_factory.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "Title": "Talk to Your Factory: Optimizing Factory Operations with Generative AI",
+  "Summary": "Explore how Generative AI is transforming factory operations with this end-to-end demo.",
```
Comment (fcabrera23) [inline drops/talk_to_your_factory.json:3]: Overall looks good - Given that the summary is what is shown in the Drop card, i'd suggest adding some extra wording explaining what the e2e demo does. Keep ti brief, but giving a bit more insights to the user may be helpful

**Review by fcabrera23 — CHANGES_REQUESTED** [review]
Added a minor suggestion

File: drops/talk_to_your_factory.json (hunk 3 above)
Comment (chriscrcodes) [reply drops/talk_to_your_factory.json]: Done, thanks.

**Review by fcabrera23 — CHANGES_REQUESTED** [review]
image

Please make it briefer - Adding image on how its looking right now

**Comment by chriscrcodes** [comment]
Done.

**Review by fcabrera23 — APPROVED** [review]
LGTM

**Pull request merged**